docker/almalinux-image.docker

```diff
@@ -20,17 +19,21 @@ RUN yum -y install \
    sudo \
    python3-pip
 
-RUN until curl -S -s --retry 9 --retry-connrefused --retry-delay 1 http://mirror.centos.org/centos/7/os/x86_64/Packages/ant-junit-1.9.4-2.el7.noarch.rpm -o ant-junit-1.9.4-2.el7.noarch.rpm ; do echo "curl failed… trying again… " ; done
-
-RUN rpm -i --nodeps ant-junit-1.9.4-2.el7.noarch.rpm
-
 RUN pip3 install --upgrade pip
 
 # install Sphinx to generate docs
 RUN pip3 install \
    Sphinx \
    sphinx_rtd_theme
 
+# install ant
+RUN mkdir /usr/share/ant && \
+    wget --tries 9 https://dlcdn.apache.org/ant/binaries/apache-ant-1.10.12-bin.tar.gz && \
```

Comment:
is this "tries" really needed? The default, according to man, is 20. Do you lower this to 9 on purpose?

You can also download sha512 file (1), do hash of what you downloaded and compare it 

(1) https://dlcdn.apache.org/ant/binaries/apache-ant-1.10.12-bin.tar.gz.sha512